static/docs/user-guide/contributing-documentation.md

```diff
@@ -105,12 +80,13 @@ follow the steps below:
 - **Markdown and JS files:** content must be properly formatted at 80 symbols
   width. We recommend using Visual Studio Code and the
   [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap)
-  plugin to format the content of Markdown files or install the pre-commit hook
-  described above.
+  plugin to format the content of Markdown files or it will be done by a
+  pre-commit hook integrated.
```

Comment:
can we specify here and in the dev environment that we need it to run the hook. It won't work w/o `npm install` at least, right?

Reply:
@shcheklein Okay, I added it here. But where exactly does it go it development environment?

Reply:
Probably in the paragraph before the `npm install`. We can split it, and most of it that is related to JS code changes move before the next step - running the server.